content/updates/2020-09/1-flat-files-exports-with-labels.md

```diff
@@ -0,0 +1,27 @@
+---
+pim_announcement_img: ./img/Product-export-profile.png
+pim_announcement_alt_img: "Flat files exports with labels"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Display labels instead of codes in your product exports
+::: meta-data type="New" features="Collaboration, Productivity" available="mid-September" link-to-doc="../articles/exports.html#update-an-export-profile"
+:::
+
+To save time and improve the reading of exported files, we added an option in our XLSX and CSV export functionalities to display attribute labels instead of codes in the exported files.
```

Comment:
```suggestion
To save time and improve the reading of exported files, we added an option in our XLSX and CSV export functionalities to display attribute labels instead of codes in the exported files.  
```